src/components/Asset/Edit/AddService.tsx

```diff
@@ -160,6 +163,12 @@ export default function AddService({
         newFiles = filesEncrypted
       }
 
+      const credentials = generateCredentials(
+        undefined,
```

Comment:
Usually optional parameters should be last

Reply:
I am thinking what to do about it, but perhaps it's best to just keep it like that. It is used like that even in the code that I haven't changed e.g. here https://github.com/OceanProtocolEnterprise/market/blob/main/src/components/Publish/_utils.ts#L242
LMK what you think